make/photon/prepare/utils/cert.py

```diff
@@ -99,33 +100,34 @@ def prepare_registry_ca(
         os.chown(private_key_pem_path, DEFAULT_UID, DEFAULT_GID)
 
 
-def prepare_trust_ca(**kwargs):
-    def f(path: str, file_name: str):
+def prepare_trust_ca(config_dict):
+    if shared_cert_dir.exists():
```

Comment:
could you pring some log message when copying cert to the shared cert directory?